File: src/main/java/com/neo/caption/ocr/property/CorsProperties.java
package com.neo.caption.ocr.property;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties(prefix = "cors")
public class CorsProperties {

    private String[] allowedOriginPatterns;
    private String allowedHeader;
    private String allowedMethods;

}

File: src/main/kotlin/com/neo/caption/ocr/property/CorsProperties.kt
package com.neo.caption.ocr.property

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "cocr.cors")
data class CorsProperties(
    val originPatterns: List<String>,
    val allowedHeader: List<String>,
    val allowedMethods: List<String>
)
